src/compose/ComposeBox.js

```diff
@@ -261,6 +277,11 @@ class ComposeBox extends PureComponent<Props, State> {
     dispatch(addToOutbox(this.getDestinationNarrow(), message));
 
     this.setMessageInputValue('');
+
+    if (this.mentionWarnings) {
+      this.mentionWarnings.current.getWrappedInstance().clearMentionWarnings();
```

Comment:
This condition still says just `this.mentionWarnings` while the other says `this.mentionWarnings.current`, and after the method call inside this one was edited (following https://github.com/zulip/zulip-mobile/pull/4101#discussion_r463396159 ) to use `.current`.

Again they should probably all match -- they should if there isn't a reason for them to vary.

But also, if the code works in this version, then that means the condition isn't actually needed at all, because the condition in this version isn't doing anything. Perhaps just take these conditions out? They don't appear in the examples in the docs:
https://reactjs.org/docs/refs-and-the-dom.html

Reply:
The Flow typedef seems to prefer the use of `current`:
```js
  declare export function createRef<T>(
  ): {|current: null | T|};
```

Reply:
And the Docs say we should access a ref using `current`:
https://reactjs.org/docs/refs-and-the-dom.html#accessing-refs

I'll use current in all places for now. If you still think we should leave it out, let me know. I'll remove it then.